### PR TITLE
Added starter script for scraping n problems according to varying dif…

### LINKED ISSUE #753: LeetCode Scrapper
# Details

Come up with a script that scraps `n` number of problems from `any category/difficulty` provided by the user and store all the `n` problems separately in PDFs. The PDF file should contain the problem title, statement, and test cases.

User Input: `n` and `category/difficulty` of problem
Expected Output: `n` PDF files

### PR DESCRIPTION
**Pull request by AshuKV**
…ficulty

# Description

Please include a summary of the change and which issue is fixed. List any dependencies that are required for this change.

## Fixes #753 

Replace `issue_no` with the issue number which is fixed in this PR

## Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/CONTRIBUTING.md)?

- [x] Yes
- [ ] No

## Type of change

_Please delete options that are not relevant._

- [ ] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update
- [ ] Documentation Update

## Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have created a helpful and easy to understand `README.md`
- [x] My documentation follows [`Template for README.md`](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md)
- [x] My changes generate no new warnings
- [ ] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works.

### CODE REVIEW

File: LeetCode-Scrapper/format.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+f1 = open('hard','r')
```
Comment (kaustubhgupta) [inline LeetCode-Scrapper/format.py:1]: What is this file?

File: LeetCode-Scrapper/ques.py [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+	links = getLinks()
+	print( len(links) )
+	for l in links:
+		print( l )
+	print('\n')
+	total = len(links)
+	ind = 296
+	links = links[296:]
+	files = {}
+	files['Easy'] = open('easy','a')
```
Comment (kaustubhgupta) [inline LeetCode-Scrapper/ques.py:50]: Files without extensions?

File: LeetCode-Scrapper/ques.py [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+			if meta.get('name') == 'description':
+				ques = meta.get('content')
+				ques = ques.encode('utf-8')
+				files[diff].write(str(count[diff]) + ') ' + name + '\n')
+				files[diff].write(ques.decode("utf-8"))
+				count[diff] += 1
+				break
+		ind +=1
+
+	print("done: ",ind)
```
Comment (kaustubhgupta) [inline LeetCode-Scrapper/ques.py:95]: I have mentioned in the script as well that each question needs to be stored as a single PDF but there is no code for that

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]
what are links and not found? Both files have no extension

**Comment by kaustubhgupta** [comment]
@AshuKV You need implement this in the same way as it's done here: https://github.com/avinashkranjan/Amazing-Python-Scripts/pull/837

**Review by AshuKV — COMMENTED** [review]
I have reviewed the changes

**Comment by AshuKV** [comment]
I have [committed](https://github.com/avinashkranjan/Amazing-Python-Scripts/pull/833/commits/068d694593cc114f16b397b8abc1c47a2f67da69) the new files, please review it and merge it accordingly.

File: LeetCode-Scrapper/ques.py [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+from selenium.webdriver.common.by import By
+from selenium.common.exceptions import NoSuchElementException
+from selenium.common.exceptions import TimeoutException
+from fpdf import FPDF
+
+
+capa = DesiredCapabilities.CHROME
+capa["pageLoadStrategy"] = "none"
+
+driver = webdriver.Chrome("C:/Softwares/chromedriver_win32/chromedriver")
```
Comment (kaustubhgupta) [inline LeetCode-Scrapper/ques.py:17]: Take input for chrome driver path

**Review by AshuKV — COMMENTED** [review]
I have reviewed the files. You can merge the PR now.

File: LeetCode-Scrapper/ques.py [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.common.by import By
+from selenium.common.exceptions import NoSuchElementException
+from selenium.common.exceptions import TimeoutException
+from fpdf import FPDF
+
+
+capa = DesiredCapabilities.CHROME
+capa["pageLoadStrategy"] = "none"
+
```
Comment (kaustubhgupta) [inline LeetCode-Scrapper/ques.py:16]: Where is the input for chrome driver path?

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

**Comment by kaustubhgupta** [comment]
@AshuKV?

**Comment by AshuKV** [comment]
Hey @kaustubhgupta I would be pushing the code by EOD today

**Review by AshuKV — COMMENTED** [review]
Pls confirm that adding path in this manner works.

File: LeetCode-Scrapper/ques.py [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+# Input for chrome driver path
+chromeDriverPath=  "C:\chromedriver_win32\chromedriver.exe";
```
Comment (kaustubhgupta) [inline LeetCode-Scrapper/ques.py:18]: When I say take input for chrome driver path, I mean to say let the user enter the path of their chrome driver via the terminal. use input function

**Review by kaustubhgupta — COMMENTED** [review]
Your code is way to similar to one merged earlier for CodeChef scrapping: https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Codechef%20Scrapper/codechef.py

@santushtisharma10 @antrikshmisri

**Comment by santushtisharma10** [comment]
Yes @kaustubhgupta , even the comments and function names are exactly same.

**Comment by kaustubhgupta** [comment]
PR closed and issue unassigned

**Pull request closed without merging**